CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixed
 * Partition-Based to Flexible Sync Migration for migrating a client app that uses partition based sync to use flexible sync under the hood if the server has been migrated to flexible sync is officially supported with this release. Any clients using an older version of Realm will receive a "switch to flexible sync" error message when trying to sync with the app. ([realm/realm-core#6554](https://github.com/realm/realm-core/issues/6554), since v11.9.0)
+* Calling `snapshot()` on an array of primitive types will product undefined results, and the method has been removed.
```

Comment:
The `snapshot` method hasn't been removed? Also, we should use "list" instead of "array" (as that's core terminology):

```suggestion
* Calling `snapshot()` on a list of primitive types is not supported and now throws.
```

Reply:
"... on a `Realm.List` of ..."?